packages/wells/src/client/model/Wellbore.ts

```diff
@@ -58,4 +59,11 @@ export interface Wellbore {
    * @memberof Wellbore
    */
   getWellHeadLocation(): Promise<WellHead | undefined>;
+  /**
+   * @type {Promise<Asset[]>}
+   * @memberof Wellbore
+   * If the source parameter is set, it will only return source assets from that source system.
+   * The source parameter can for example be EDM, Diskos, Openworks, etc.
+   */
```

Comment:
nit: If any, we have generally added text on top of the docstring and not one the bottom